### PR TITLE
Added $onlyif_empty to db.pp to facilitate once-only import of sql datab...

### PR DESCRIPTION
**Pull request by gibbsoft**
...ase file, rather than import at every puppet run.

I modified db.pp to conditionally check if there are any existing tables in the database, if the schema already has tables then we may not wish to import the specified sql flat file.

### CODE REVIEW

File: manifests/db.pp [hunk 1]
```diff
@@ -67,6 +68,10 @@
         require     => Mysql_grant["${user}@${host}/${table}"],
         subscribe   => Mysql_database[$dbname],
         timeout     => $import_timeout,
+        onlyif      => $onlyif_empty ? {
+          true    => "mysql ${dbname} -e 'show tables' |wc -l | expr 1 /",
+          default => "true",
+        }
```
Comment (igalic) [inline manifests/db.pp]: this shouldn't be quoted

**Comment by gibbsoft** [comment]
I realised that I'd made a mistake, so rolled out my commit and replaced it with another.  The onlyif command now uses 'test'.  The default of "true" is intentionally quoted as it is a call to /bin/true to derive a return value of 0.

File: manifests/db.pp [hunk 2]
```diff
@@ -67,6 +68,10 @@
         require     => Mysql_grant["${user}@${host}/${table}"],
         subscribe   => Mysql_database[$dbname],
         timeout     => $import_timeout,
+        onlyif      => $onlyif_empty ? {
+          true    => "test $(mysql ${dbname} -e 'show tables' |wc -l ) -eq 0",
+          default => "false",
```
Comment (igalic) [inline manifests/db.pp]: the reason tests fail on Travis is this line. you're "quoting" a boolean value.

Comment (gibbsoft) [inline manifests/db.pp]: It's a false positive: I'm not really quoting a boolean value - it's a call to /bin/false.

Comment (gibbsoft) [inline manifests/db.pp]: I've wrapped in in 'test' instead, I'll close this pull-request and raise another against the new commit.

**Pull request closed without merging**